### DISCUSSION TITLE
m7a启动异常

### DISCUSSION THREAD

**Discussion by Xiafeng-maples** [moesnow/March7thAssistant#371] [General]
m7a2.54最新  系统win11 23h2,已知在22h2也有这种状况
正常点击m7a后任务开始运行，但是会一直挂在后台当后台进程，过两三分钟后才弹出窗口转到应用，期间任务栏没有图标，只能在任务管理器翻到，试了试也不是扩展到第二块屏幕的问题，win+tab也没有。重装过也没解决
不知道你们有没有这种情况

**Comment by Xiafeng-maples** [comment]
管理员启动也没区别

**Comment by shotamiao** [comment]
我也遇到过，未知原因导致我前几天又能正常使用，但现在又不能正常使用，我尝试重新下载一个全新的后可能解决了（不确定）

**Reply by Xiafeng-maples** [reply]
我现在这非常稳定，重装也没用（恼。每次都要等好一会

**Reply by shotamiao** [reply]
> 我现在这非常稳定，重装也没用（恼。每次都要等好一会

我现在的情况也是一样

**Comment by shotamiao** [comment]
疑似原因![Screenshot_20241109-212024_ToDesk.jpg](https://github.com/user-attachments/assets/4c343fbf-2937-4c15-9c69-34b83ce98b8e)

当我打开三月七小助手时微软杀毒占用CPU极高
而不打开三月七小助手时占用较低![Screenshot_20241109-212825_ToDesk.jpg](https://github.com/user-attachments/assets/3084b4ed-d1a8-4fac-9abd-003bef6b7a13)

**Reply by shotamiao** [reply]
![Screenshot_20241109-213135_ToDesk.jpg](https://github.com/user-attachments/assets/9cab77ae-1607-4ad0-b80d-edaf8140b00a)

**Comment by shotamiao** [comment]
后续：在微软杀毒中，将三月七小助手文件夹加入排除项就解决了

**Reply by Xiafeng-maples** [reply]
神医啊。我这里虽然没有突然负载飙升，不过加白名单后确实能直接打开了
